### ISSUE TITLE
Switching portrait to landscape mode (or vice versa) fails

### ISSUE THREAD

**Issue by dickerpulli** [antonKalinin/react-native-image-view#15]
- I start my app in landscape mode
- the component containing the ImageView is mounted 
- then I switch to portrait mode
- I open the ImageView
---> The overlay is displayed incorrectly
---> I think the ImageView stores the initial screen dimensions and doesn't recognize a later change of the dimensions due to a switch from portrait to landscape mode

**Comment by antonKalinin** [comment]
@dickerpulli You are right, this place needs improvement. PRs are welcome) I hope I will have time this weekend to work around react-native-image-view
